ee/codegen/src/__test__/generate-code.test.ts

```diff
@@ -0,0 +1,72 @@
+import fs from "fs";
+import os from "os";
+import path, { join } from "path";
+
+import { WorkflowProjectGenerator } from "src/project";
+import { getAllFilesInDir } from "src/utils/files";
+
+const moduleName = "testing";
+const vellumApiKey = "<TEST_API_KEY>";
+const fixturesDir = path.join(__dirname, "generate-code-fixtures");
+
+/**
+ * New pattern for testing project level codegen features and to start reducing the size of `projecte.test.ts`.
+ * - Create a new file within `__test__/generate-code-fixtures`
+ * - The file should export a default object with the following properties:
+ *   - assertions: an array of file paths to assert snapshots against
+ *   - workflow_raw_data: the raw workflow data to pass to the `WorkflowProjectGenerator`
+ *   - The rest of workflow version exec config data also top level.
+ */
+describe("generateCode", () => {
```

Comment:
cc: @vincent0426 @alex-nork @NgoHarrison for thoughts here to start spreading out the use of `project.test.ts`